sdk/core/abort-controller/package.json

```diff
@@ -57,7 +57,7 @@
   ],
   "scripts": {
     "build:samples": "echo Obsolete",
-    "build:test": "npm run clean && tshy && dev-tool run build-test",
+    "build:test": "echo skipped. actual commands inlined in browser test scripts",
```

Comment:
another approach is to have `build:test:node` and `build:test:browser` but it seems that we used to have that but moved away from it.